app/test_main.py

```diff
@@ -1,3 +1,34 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    ("cat_years", "dog_years", "result"),
```

Comment:
Update without brackets
```suggestion
    "cat_years, dog_years, result",
```

[LINK](https://docs.pytest.org/en/stable/example/parametrize.html#paramexamples)

![image](https://github.com/user-attachments/assets/7dd0689e-828e-4225-a68a-f30b07061b1a)